content/_partners/state-and-local._en.md

```diff
@@ -1,66 +1,66 @@
+  Crises, like the COVID-19 global pandemic and ongoing natural disasters, make it more urgent than ever for the public to gain fast and easy access to their benefits and services. The General Services Administration’s (GSA) Technology Transformation Services (TTS) realizes the burden this has put on state and local governments. Our team, with deep expertise in security, product, engineering, and user experience, is seeking state, local, and territory government partners that want to take advantage of all that Login.gov has to offer.
 ssa_quote: |-
-    > “The SSA UXG team appreciates the willingness of the Login.gov team to listen to our goals, concerns, and questions, then work with us to identify a path forward that will benefit the customers (end users) of SSA and other federal agencies. The team’s flexibility and willingness to hear our suggestions is very much appreciated. We enjoy brainstorming together and sharing research findings as we try to help each other move forward.”
-    {:class="margin-0"}
+  > “The SSA UXG team appreciates the willingness of the Login.gov team to listen to our goals, concerns, and questions, then work with us to identify a path forward that will benefit the customers (end users) of SSA and other federal agencies. The team’s flexibility and willingness to hear our suggestions is very much appreciated. We enjoy brainstorming together and sharing research findings as we try to help each other move forward.”
+  {:class="margin-0"}
 
-    Elizabeth Hanst
-    Technology Lead User Experience Specialist
-    Social Security Administration
+  Elizabeth Hanst
+  Former Technology Lead User Experience Specialist
+  Social Security Administration
 partnership_steps_title: >-
-    ## Steps to partnership
+  ## Steps to partnership
 partnership_steps_header1: >-
-    Introductions and determine compatibility
+  Introductions and determine compatibility
 partnership_steps_body1: >-
-    <p style="display: inline">During the introduction call, an account manager will walk through Login.gov services and answer any questions that you have. This step will determine if Login.gov is a good fit for your agency.</p>
+  <p style="display: inline">During the introduction call, an account manager will walk through Login.gov services and answer any questions that you have. This step will determine if Login.gov is a good fit for your agency. [Contact our Partnerships Team to get started](/partners/contact/){:class="external-link"}.</p>
```

Comment:
**issue:** Nothing shows up for this link in the test build, see screenshot
<img width="730" alt="Screen Shot 2022-08-04 at 2 35 05 PM" src="https://user-images.githubusercontent.com/51330839/182957008-e62c60e5-c54c-43c1-b353-15654859aa06.png">
.

Reply:
Yeah I was torn between whether to use that syntax or the <a> tag, I guess maybe its better to use the <a> tag

Reply:
OK it looks like that field wasn't supporting markdown because of it was missing the `markdownify` property in the html.. and that was where that other link was getting added too.